linker/NetCore.props

```diff
@@ -1,13 +1,13 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFrameworks>netcoreapp2.0;net46</TargetFrameworks>
     <RuntimeFrameworkVersion>2.0.0-beta-001509-00</RuntimeFrameworkVersion>
     <DefineConstants>$(DefineConstants);NET_CORE</DefineConstants>
```

Comment:
I see.  That's unfortunate naming.  I would at least put a comment there for now, or rename the define if its not too much trouble.  Something like "ILLINK" or "FEATURE_ILLINK" etc.